### DISCUSSION TITLE
Problem add to cart product page

### DISCUSSION THREAD

**Discussion by mircomartin** [EvgeniyMukhamedjanov/liquid-ajax-cart#61] [General]
@EvgeniyMukhamedjanov Hello!

I need your help, i need to configure for two options.

1) When added to cart popup is active.
For this i need when customer click in add to cart button, the cart-drawer, nop open(only in product page).
2) When added to cart popup is disabled
Open normally the cart-drawer.

How can i do that because now when popup is active the drawer open above popup.

Also every-time that click in any button the product added to the cart not only with add to cart button

You can test it

https://leadwaysdk.myshopify.com/

pass "skaure"

**Comment by EvgeniyMukhamedjanov** [comment]
Hello,
You should use the [subscribeToCartAjaxRequests](https://liquid-ajax-cart.js.org/reference/subscribeToCartAjaxRequests/) function, where you can perform anything you want when a product is added to the cart.
```
import { subscribeToCartAjaxRequests } from '{{ "liquid-ajax-cart-v1.10.3.js" | asset_url }}';

subscribeToCartAjaxRequests(( requestState, subscribeToResult ) => {
  if ( requestState.requestType === 'add' ) {
    subscribeToResult( requestState => {
      // a product just added to the cart 
      // perform your actions here
